source/rules/roles.txt

```diff
@@ -266,231 +323,153 @@ the order in which they are evaluated and applied. Each role's apply
 
-         .. step:: Create a New Role
-
-            To create a new role, click :guilabel:`Add role`. You can
-            either start from scratch or use a preset role as a starting
-            point.
-
-         .. step:: Configure the Role
-
-            A. Give the role a name. The name can be anything you want but
-               must be unique within a given collection. Consider using names
-               that describe the user (e.g. ``Admin``) and/or their
-               relationship to the data (e.g. ``Owner``).
-
-            #. Define the Role's :guilabel:`Apply When` Condition. 
-               A role's :ref:`Apply when <apply-when-expressions>` condition 
-               determines when a given user has the role for a given document.
-
-            #. Define the role's :guilabel:`Document Permissions`. For more 
-               information, see :ref:`document-level-permissions`. 
-
-            #. Define the role's :guilabel:`Field Permissions`. For more 
-               information, see :ref:`field-level-permissions`. 
-
-         .. step:: Save the Role
-
-            Save the role. Once you have finished defining roles for the collection, 
-            App Services will immediately begin using the roles you
-            defined for all incoming queries on the collection.
-
-            .. note::
+      1. Click :guilabel:`Rules` in the left navigation menu and then select a
+         collection from the data source's collection list. You can also select
+         :guilabel:`Default roles and filters` to configure default rules for
+         the entire cluster.
+      
+         If no roles are defined yet, you'll be prompted to create a new
+         one. Otherwise you'll see an ordered list of existing roles.
+         
+      #. To create a new role, click :guilabel:`Add role`. You can
+         either start from scratch or use a preset role as a starting
+         point.
+         
+      #. Give the role a name. The name can be anything you want but
+         must be unique within a given collection. Consider using names
+         that describe the user (e.g. ``Admin``) and/or their
+         relationship to the data (e.g. ``Owner``).
+      
+      #. Define an "apply when" expression that determines when a given user has
+         the role for a given document.
+      
+      #. Define :ref:`document filters <document-filters-expressions>` for the
+         role.
 
-               If a collection has more than one role assigned, you can modify their 
-               :ref:`role order <role-order>` by clicking the arrows on each role.
+      #. Define :ref:`document-level permissions <document-level-permissions>`
+         for the role.
+      
+      #. Define :ref:`field-level permissions <field-level-permissions>` for the
+         role.
+      
+      #. Save the role.
 
+      #. If a collection has more than one role assigned, you can modify
+         their :ref:`role order <role-order>` by clicking the arrows on
```

Comment:
Slight semantic change to make the connection to collection clearer. Otherwise, it may seem like the roles themselves have a role order.
```suggestion
         the :ref:`role order <role-order>` by clicking the arrows on
```